### PR TITLE
Facebook profile URL fallback

### PR DESCRIPTION
**Pull request by bologer**
| Q                        | A
| ------------------------ | ---
| Fixed Issues?            | No
| Patch: Bug Fix?          | Kind of
| Major: Breaking Change?  | No
| Minor: New Feature?      | Yes
| Documentation PR         | No

Implemented fallback for Facebook to always have profile URL set on the identity user returned from the API. After sometime, Facebook stopped returning `link` from the API, so that the profile URL became empty.

### CODE REVIEW

File: src/Provider/Facebook.php [hunk 1]
```diff
@@ -87,22 +92,29 @@ public function getUserProfile()
...
+        $userProfile->profileURL = $data->get('link');
+        $userProfile->webSiteURL = $data->get('website');
+        $userProfile->gender = $data->get('gender');
+        $userProfile->language = $data->get('locale');
         $userProfile->description = $data->get('about');
-        $userProfile->email       = $data->get('email');
+        $userProfile->email = $data->get('email');
+
+        /**
+         * Fallback for profile URL in case Facebook does not provide "pretty" link with username (if user set it).
```
Comment (ApacheEx) [inline src/Provider/Facebook.php]: let's use inline style of comment

File: src/Provider/Facebook.php [hunk 2]
```diff
@@ -343,15 +355,30 @@ protected function fetchUserActivity($item)
...
         return $userActivity;
     }
+
+    /**
+     * Get profile URL.
+     *
+     * @param int $identity User ID.
+     * @return string|null NULL when identity is not provided.
+     */
+    public function getProfileUrl($identity)
```
Comment (ApacheEx) [inline src/Provider/Facebook.php]: or you make it as protected or you move it upper to have right order

**Comment by bologer** [comment]
@ApacheEx done! :+1:

**Comment by ApacheEx** [comment]
great, thank you 💯

**Pull request merged**